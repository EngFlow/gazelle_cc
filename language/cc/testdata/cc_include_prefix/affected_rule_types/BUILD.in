# gazelle:cc_include_prefix my_prefix
# gazelle:cc_strip_include_prefix /affected_rule_types
