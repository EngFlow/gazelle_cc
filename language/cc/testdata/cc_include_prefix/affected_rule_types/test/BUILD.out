load("@rules_cc//cc:defs.bzl", "cc_test")

cc_test(
    name = "test",
    srcs = ["test.cc"],
)
